From 39772585033c1c0aff96f297e93de7be53f9b114 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sat, 23 Mar 2019 01:17:11 +0100
Subject: [PATCH] tests: Fix build with Qt 5.13

QTest::toString(QModelIndex) was added in
7ef0b575b38d267bd3dc14ff46935d556562ff00 and thus causes a build
error with Qt 5.13 because it's redefined here.

Change-Id: I015800e49cf8d0e87f3541642406396f150d0eeb
---
 tests/Imap/test_Imap_BodyParts.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/Imap/test_Imap_BodyParts.cpp b/tests/Imap/test_Imap_BodyParts.cpp
index f1c577fa..b6e0421d 100644
--- a/tests/Imap/test_Imap_BodyParts.cpp
+++ b/tests/Imap/test_Imap_BodyParts.cpp
@@ -51,6 +51,7 @@ struct Data {
 
 Q_DECLARE_METATYPE(QList<Data>)
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 13, 0)
 namespace QTest {
 template <>
 char *toString(const QModelIndex &index)
@@ -60,6 +61,7 @@ char *toString(const QModelIndex &index)
     return qstrdup(buf.toUtf8().constData());
 }
 }
+#endif
 
 using namespace Imap::Mailbox;
 
